pkgs/applications/networking/mailreaders/thunderbird/default.nix

```diff
@@ -7,7 +7,7 @@
 , autoconf213, which, m4
 , writeScript, xidel, common-updater-scripts, coreutils, gnused, gnugrep, curl
 , enableGTK3 ? false, gtk3, wrapGAppsHook
-, debugBuild ? false
+, debugBuild ? false, config
```

Comment:
Please don't use `config` in a derivation, instead make a new flag argument (`enableCalendar` or something like that). You can set this flag from `config` in `all-packages.nix`. Personally I think that `config` shouldn't be used for single-package options.